roles/ceph-common/defaults/main.yml

```diff
@@ -302,6 +306,15 @@ radosgw_keystone_token_cache_size: 10000
 radosgw_keystone_revocation_internal: 900
 radosgw_s3_auth_use_keystone: "true"
 radosgw_nss_db_path: /var/lib/ceph/radosgw/ceph-radosgw.{{ ansible_hostname }}/nss
+# Settings for the RGW usage logging described at http://docs.ceph.com/docs/jewel/man/8/radosgw/#usage-logging
+radosgw_usage_log: false
+radosgw_usage_log_tick_interval: 30
```

Comment:
I suspect defining these in ``[rgw]`` would set those baseline defaults for all rgw nodes, but I don't know that for certain.

Your approach does have the advantage of exposing these config options to the user, but I know that @leseb has been pushing to reduce the amount of config variables like this and promote usage of ``ceph_config_overrides``.